src/app/screens/Login/index.js

```diff
@@ -1,14 +1,50 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 
 import Login from './layout';
 
-// TODO: add validations and submit
+import { emailRegex } from '../../../constants/regex';
+import { Home } from '../../../constants/routes';
 
-class LoginContainer extends PureComponent {
+class LoginContainer extends Component {
+    
+    state = { invalidEmail: false, email: '' , invalidPassword: false, password: '' };
+
+    onChangeEmail = email => this.setState({email:email});
+
+    onChangePassword = password => this.setState({password:password});
+
+    validateEmail = () => {
+        if(emailRegex.test(this.state.email)) {
+            this.setState({invalidEmail: false});
+        } else {
+            this.setState({invalidEmail: true});
```

Comment:
first destruct `email` from state and then:
`this.setState({ invalidEmail: !emailRegex.text(email) });`